annotation/app/models.py

```diff
@@ -97,6 +102,13 @@ class AnnotatedDoc(Base):
     tasks = relationship("ManualAnnotationTask", back_populates="docs")
 
 
+def default_tree(column_name: str) -> Callable:
```

Comment:
May you specify callable args and return type? Strange that mypy doesn't complain about it

Reply:
1st arg can not be specified because it can not be imported